src/prompts.js

```diff
@@ -21,6 +21,12 @@ const config = [
     choices: [
       { name: ':smile:', value: 'code' }, { name: '😄', value: 'emoji' }
     ]
+  },
+  {
+    name: constants.SIGNED_COMMIT,
+    message: 'Choose default answer for Signed commit',
+    type: 'list',
```

Comment:
Why you don't use type `confirm`? You'll get a boolean (true/false) that it's easier to use and makes more sense! If you think it, you can sign or not sign a commit so that's a boolean